packages/aws-rfdk/lib/deadline/lib/worker-configuration.ts

```diff
@@ -29,6 +29,26 @@ import {
   Version,
 } from './version';
 
+/**
+ * Provider for adding user data scripts
+ */
+export interface IInstanceUserDataProvider {
+  // Method that is invoked before configuration Cloud Watch Agent.
+  preCloudWatchAgent(host: IHost): void;
```

Comment:
these should be made into proper TSDoc strings so that they are visible in our API docs and discoverable from IDEs.

Reply:
This was missed.

Reply:
Unless I'm wrong here, the `//` syntax is just a comment and not a docstring. i.e. it doesn't end up in the API documentation.